src/ProductConstructionService/ProductConstructionService.Api/Program.cs

```diff
@@ -14,9 +14,9 @@
 bool isDevelopment = builder.Environment.IsDevelopment();
 bool useSwagger = isDevelopment;
 
-builder.ConfigurePcs(
+await builder.ConfigurePcs(
     addKeyVault: true,
-    initializeService: !isDevelopment,
+    addRedis: true,
```

Comment:
There was only one such flag, so I just added another here. But maybe we want to unify them under some "don't add authenticated resources" flag as it seems to be only used for tests?